Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,101 @@
+
+        <figcaption>
+            An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+            uncertain how the trait evolved.</figcaption>
+
+    </figure>
+
+    <p> To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus
+        pulling
+        two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like
+        nesting
+        bowls — and carts them away.</p>
+
+    <blockquote>“The octopus is carrying these tools around because it has some understanding they may be useful in
+        the
+        future,” said Mr. Amodio</blockquote>
+
+    <p>But his amazement is mixed with puzzlement.</p>
+
+    <p>For decades, researchers have studied how certain animals evolved to be intelligent, amoung them apes,
+        elephants,
+        dolphins and even some birds, such as crows and parrots.</p>
+
+    <p>But all the scientific theories fail when it comes to cephalopods, a group that includes octopuses, squid
+        and cuttlefish. Despite feats of creativity, they
+        <a href=<cite>lack some hallmarks</cite></a>
+        of intelligence seen in other species.
+
+        <p>“It’s an apparent paradox that’s been largely overlooked in the past,” said Mr. Amodio. He and five
+            other
+            experts on animal intelligence explore this paradox in a paper published this month in the journal
+            Trends
+            in Ecology and Evolution.</p>
+
+        <aside>
+            <blockquote>
+                “It’s an apparent paradox that’s been largely overlooked in the past,” said Mr. Amodio.
+            </blockquote>
+        </aside>
+
+
+        <p>For scientists who study animal behavior, intelligence is not about acing a calculus test or taking a
+            car
+            apart and putting it back together. Intelligence comprises sophisticated cognitive skills that help an
+            animal thrive.</p>
+
+        <figure>
+
+            <img src="Images/Unknown.jpeg" width="1000" height="500" alt="octupus with tnetacles floating in the water">
+            <figcaption>The plural of "Octupus" is "Octopi" although, this is one singular Octupus.</figcaption>
+        </figure>
+
+        <p>That may include the ability to come up with solutions to the problem of finding food, for example, or
+            a knack for planning for some challenge in the future. Intelligent animals don’t rely on fixed
+            responses to
+            survive — they can invent new behaviors on the fly.</p>
+
+        <p>To measure animal intelligence, scientists observe creatures in the wild — watching a dolphin stick
+            a sponge on its beak to avoid getting cuts from sharp they bring animals into the lab
+            and
+            offer them puzzles to solve, such as rewarding crows when they <a href=<cite>learn to rip paper into
```

Comment:
Same issue as noted above:

```suggestion
            offer them puzzles to solve, such as rewarding crows when they <a href="https://www.nytimes.com/2018/11/30/science/animal-intelligence-octopus-cephalopods.html">learn to rip paper into
```